services/repology/repology-repositories.service.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { nonNegativeInteger } = require('../validators')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: nonNegativeInteger,
+}).required()
+
+module.exports = class RepologyRepositories extends BaseSvgScrapingService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'repology/repositories',
+      pattern: ':projectName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Repology - Repositories',
+        namedParams: { projectName: 'starship' },
+        staticPreview: this.render({ repositoryCount: '18' }),
+        keywords: ['repositories'],
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'in repositories' }
+  }
+
+  static render({ repositoryCount }) {
+    return {
+      message: repositoryCount,
```

Comment:
By any chance do you know what the realistic ceiling would be for values we could see? Shields uses a common set of logic for metric/numerical values in the rendering (for example 1,045 would be rendered as 1k) and I'm thinking it'd probably be best to go ahead and use taht same formatter here.

Similar to the validators for schemas, we've got a lot of helpers for various formatting scenarios. I believe you'd want to use `metric` here from https://github.com/badges/shields/blob/master/services/text-formatters.js

```suggestion
      message: metric(repositoryCount),
```

Reply:
The most I've seen is 55, but I think it'd best to be safe and use it. 👍